app/views/shared/buttons/_ab_form.html.haml

```diff
@@ -22,14 +22,14 @@
 
   #custom_button_tabs
     %ul.nav.nav-tabs
-      = miq_tab_header('ab_options_tab', @sb[:active_tab]) do
```

Comment:
@ZitaNemeckova removing `@sb[:active_tab]` might flip the tab to first tab when a transaction to server is sent up to re-populate fields, maybe when changing expressions.

Reply:
You are right. Removing the offending commit.